specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-02-01-preview/mfe.json

```diff
@@ -1506,7 +1506,7 @@
         },
         "x-ms-long-running-operation": true,
         "x-ms-long-running-operation-options": {
```

Comment:
This is not allowed and is flagged in the tests : 

[https://github.com/Azure/azure-rest-api-specs/pull/23105/checks?check_run_id=12056875408](https://github.com/Azure/azure-rest-api-specs/pull/23105/checks?check_run_id=12056875408)

Reply:
Resolved based on offline thread. final-state-via should either be original-uri or it should not be specified going forward.